### PR TITLE
feat(alertmanagerConfig): render statically

### PR DESCRIPTION
**Pull request by Duologic**
Title says it, this should make the library easier to use and inspect.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -5,9 +5,14 @@
 # https://github.com/CertainLach/jrsonnet/issues/108
 JSONNET_BIN ?= jrsonnet
 
+generate:
+	jrsonnet -J generator/vendor -S -c -m alertmanagerConfig/ generator/main.jsonnet
```
Comment (jvrplmlmn) [inline Makefile]:
```suggestion
	${JSONNET_BIN} -J generator/vendor -S -c -m alertmanagerConfig/ generator/main.jsonnet
```

File: generator/main.jsonnet [hunk 2]
```diff
@@ -0,0 +1,57 @@
+local a = import 'github.com/crdsonnet/astsonnet/main.libsonnet';
+local atils = import 'github.com/crdsonnet/astsonnet/utils.libsonnet';
+local crdsonnet = import 'github.com/crdsonnet/crdsonnet/crdsonnet/main.libsonnet';
+
+local schema =
+  (import './schema.json')
+  + {
+    //'$ref': '#/$defs/Route',
```
Comment (jvrplmlmn) [inline generator/main.jsonnet]: Is this line required?

**Review by jvrplmlmn — COMMENTED** [review]
I'm not so familiar with the code generation aspect to give a meaningful review, but couple of minor things that caught my eye.

File: generator/main.jsonnet (hunk 2 above)
Comment (Duologic) [reply generator/main.jsonnet]: good point, bit of a leftover
```suggestion
```

**Pull request merged**